.github/workflows/flowzone.yml

```diff
@@ -1004,10 +1006,11 @@ jobs:
         working-directory: .
         run: tar -xvf source.tar
 
-      - uses: balena-io/deploy-to-balena-action@v0.13.0
+      - name: Deploy to ${{ matrix.balena_env.name }} 
+        uses: balena-io/deploy-to-balena-action@v0.13.0
         with:
-          balena_token: ${{ secrets.BALENA_API_KEY || secrets.BALENA_API_KEY_PUSH }}
-          environment: ${{ inputs.balena_environment }}
+          balena_token: ${{ secrets[matrix.balena_env.secret] }}
```

Comment:
I haven't been able to figure out why this doesn't work

Reply:
This works actually, is just that the flowzone repo uses `BALENA_API_KEY_PUSH` instead of `BALENA_API_KEY`